### PR TITLE
Environment

### PR DESCRIPTION
**Pull request by ondratu**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -6,6 +6,8 @@ ChangeLog
 * Python3.x support only
 * Use pylint instead of flake8
 * pre-commit config
+* EnvironFirst and EnvironLast classes which could read values
```
Comment (uzak) [inline ChangeLog]: could -> can ("could" je bud minuly cas, alebo kondicional).

File: extendparser/environ.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+class VarNameBuilder():  # pylint: disable=too-few-public-methods
+    """Support class for building variable name.
+
+    Simple changing variable name building format:
+
+        >>> from extendparser.environ import VarNameBuilder
+        >>> vnb = VarNameBuilder()
+        >>> vnb.var_format = 'PREFIX_{option}_{section}'
+        >>> vnb.varname('database', 'hostname')
+        'PREFIX_HOSTNAME_DATABASE'
```
Comment (uzak) [inline extendparser/environ.py:20]: Is `PREFIX_HOSTNAME_DATABASE` correct? Should it not be `PREFIX_DATABASE_HOSTNAME`.

File: extendparser/environ.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+class EnvironFirst(VarNameBuilder, ConfigParser):
+    """Read values from system environment first, then from ConfigParser.
+
+        >>> from os import environ
+        >>> from configparser import ConfigParser
+        >>> from extendparser.environ import EnvironFirst
+        >>> cp = EnvironFirst()
+        >>> cp.add_section("test")
+        >>> cp.getint("test", "number", fallback=1)
+        1
```
Comment (uzak) [inline extendparser/environ.py:77]: Just a note I think the `fallback` argument is usually called `default`. I'm thinking of `dict().get`. But it really doesn't matter...

**Review by uzak — COMMENTED** [review]
Nice. Finally we can do the ENV VAR work cleanly in our project.

File: extendparser/environ.py (hunk 2 above)
Comment (ondratu) [reply extendparser/environ.py:20]: That is ok, this is part of test:
```python
>>> vnb.var_format = 'PREFIX_{option}_{section}'
```

File: extendparser/environ.py (hunk 3 above)
Comment (ondratu) [reply extendparser/environ.py:77]: fallback is no my keyword argument, but argument of `configpargser.ConfigParser.getint` :-)

**Pull request merged**